index.js

```diff
@@ -579,6 +579,66 @@ module.exports = {
     });
   },
 
+  geoHashAgg: function (table, limit, precision, options, callback) {
+    var self = this;
+
+    var whereFilter, geomFilter;
+    if ( options.where ){
+      if ( options.where != '1=1'){
+        var clause = this.createWhereFromSql(options.where);
+        whereFilter = ' WHERE ' + clause;
+      } else {
+        whereFilter = ' WHERE ' + options.where;
+      } 
+    }
+
+    if ( options.geometry ){
+      var geom = this.parseGeometry( options.geometry );
+
+      if ((geom.xmin || geom.xmin === 0) && (geom.ymin || geom.ymin === 0)){
```

Comment:
if they're lat/long can't they be negative too? might want to just test that they're defined as integers/floats as truthy/falsy negative numbers can often cause issues in javascript.